lib/source-destination/source-destination.ts

```diff
@@ -286,19 +286,11 @@ export class SourceDestination extends EventEmitter {
 		throw new NotCapable();
 	}
 
-	async createReadStream(start = 0, end?: number): Promise<NodeJS.ReadableStream> {
-		return await this._createReadStream(start, end);
-	}
-
-	async _createReadStream(start = 0, end?: number): Promise<NodeJS.ReadableStream> {
+	async createReadStream(emitProgress = false, start = 0, end?: number): Promise<NodeJS.ReadableStream> {
 		throw new NotCapable();
 	}
```

Comment:
Does this only throw a new error? Same question for createSparseReadStream.
If so, why not just throw the error where due?

Reply:
This has to be defined in subclasses. The error is just here to say that this is not implemented here.